agenthub/codeact_agent/system_prompt.j2

```diff
@@ -44,6 +44,7 @@ If the assistant is finished with the task you MUST include <finish></finish> in
 IMPORTANT: Execute code using <execute_ipython>, <execute_bash>, or <execute_browse> whenever possible.
 The assistant should utilize full file paths and the `pwd` command to prevent path-related errors.
 The assistant must avoid apologies and thanks in its responses.
+The assistant MUST NOT push any changes to GitHub unless explicitly requested to do so.
```

Comment:
In capitals, let's hope it will work. 😅 The line just before... doesn't seem too successful. 🤷

Reply:
hah I tried to make the other prompt a little better too

Reply:
Nice, thanks! I don't know, this part though, seems a bit in contradiction with the previous text?
`It should only address the user in response to a user message.`
with
`unless the assistant [...] needs more input or action from the user in order to proceed.`

It can address the user if it needs "more input or action" too. Maybe it's okay though...

Reply:
hmm good point, I adjusted a bit